examples/k8petstore/k8petstore.sh

```diff
@@ -188,53 +204,80 @@ cat << EOF > slave-rc.json
   "labels": {"name": "redisslave"}
 }
 EOF
-$kubectl create -f rm.json --api-version=v1beta1
-$kubectl create -f rm-s.json --api-version=v1beta1
+$kubectl create -f rm.json --api-version=v1beta1 --namespace=$NS
+$kubectl create -f rm-s.json --api-version=v1beta1 --namespace=$NS
 sleep 3 # precaution to prevent fe from spinning up too soon.
-$kubectl create -f slave-rc.json --api-version=v1beta1
-$kubectl create -f rs-s.json --api-version=v1beta1
+$kubectl create -f slave-rc.json --api-version=v1beta1 --namespace=$NS
+$kubectl create -f rs-s.json --api-version=v1beta1 --namespace=$NS
 sleep 3 # see above comment.
-$kubectl create -f fe-rc.json --api-version=v1beta1 
-$kubectl create -f fe-s.json --api-version=v1beta1
-$kubectl create -f bps-load-gen-rc.json --api-version=v1beta1
+$kubectl create -f fe-rc.json --api-version=v1beta1 --namespace=$NS
+$kubectl create -f fe-s.json --api-version=v1beta1 --namespace=$NS
+$kubectl create -f bps-load-gen-rc.json --api-version=v1beta1 --namespace=$NS
 }
 
-function test { 
+function pollfor {
 	pass_http=0
 
 	### Test HTTP Server comes up.
 	for i in `seq 1 150`;
 	do
 	    ### Just testing that the front end comes up.  Not sure how to test total entries etc... (yet)
-	    echo "Trying curl ... $i . expect a few failures while pulling images... " 
+	    echo "Trying curl ... $PUBLIC_IP:3000 , attempt $i . expect a few failures while pulling images... " 
 	    curl "$PUBLIC_IP:3000" > result
 	    cat result
 	    cat result | grep -q "k8-bps"
 	    if [ $? -eq 0 ]; then
-		echo "TEST PASSED after $i tries !"
-		i=1000
-		break
+    		echo "TEST PASSED after $i tries !"
+	    	i=1000
+		    break
 	    else	
 		echo "the above RESULT didn't contain target string for trial $i"
 	    fi
-	    sleep 5
+	    sleep 3
 	done
 
 	if [ $i -eq 1000 ]; then
-	   pass_http=-1
+	   pass_http=1
 	fi
+    
+}
 
+function tests {
 	pass_load=0 
 
 	### Print statistics of db size, every second, until $SECONDS are up.
-	for i in `seq 1 $SECONDS`;
-	do
-	    echo "curl : $i"
-            curl "$PUBLIC_IP:3000/llen" >> result
+	for i in `seq 1 $TEST_SECONDS`;
+	 do
```

Comment:
I'm wondering if we dont fire up something like SEIGE or HTTPerf to load test the service in paralell, vs doing X curl-calls against the petstore. This would give us slightly more meaningful results as to the resiliency of the service under load and would complete in a fraction of the time.

https://registry.hub.docker.com/u/jcostello84/siege-loadtest/ - seems like a good candidate for this, and the output can be parsed for reporting elsewhere.